lib/sequelize.js

```diff
@@ -660,6 +662,14 @@ module.exports = (function() {
       type: (sql.toLowerCase().indexOf('select') === 0) ? QueryTypes.SELECT : false
     });
 
+    if (
```

Comment:
I suppose that's true.
But that also means that if you want to use CLS at all you also opt into all transactions using it.
I'm not sure it's a problem exactly though.